xbmc/interfaces/legacy/InfoTagVideo.h

```diff
@@ -565,6 +565,78 @@ namespace XBMCAddon
 #else
       String getTrailer();
 #endif
+
+#ifdef DOXYGEN_SHOULD_USE_THIS
+      ///
+      /// \ingroup python_InfoTagVideo
+      /// @brief \python_func{ getArtist() }
+      ///-----------------------------------------------------------------------
+      /// To get the artist name (for musicvideos)
+      ///
+      /// @return [string] Artist name
+      ///
+      ///
```

Comment:
can you add a 'new function' comment to each of the new methods?

```/// @python_v18 New function added.```

Reply:
Done, see 411388e84029726b6a62317703d794c1e7d74dfe